src/services/utilities.ts

```diff
@@ -2157,6 +2157,10 @@ namespace ts {
         return { text, kind: SymbolDisplayPartKind[kind] };
     }
 
+    export function functionNamePart(text: string) {

```

Comment:
this function isn't used, can you remove it?

Reply:
Oops, thanks for catching it!